src/main/java/core/basesyntax/Lottery.java

```diff
@@ -3,10 +3,9 @@
 import java.util.Random;
 
 public class Lottery {
-    public static Ball getRandomBall() {
-        ColorSupplier color = new ColorSupplier();
-        Random random = new Random();
-        int number = random.nextInt(100);
-        return new Ball(color.getRandomColor(), number);
+    public static final int NUMBER = 100;
+
+    public static Ball getRandomBall(Random random, ColorSupplier color) {
+        return new Ball(color.getRandomColor(), random.nextInt(NUMBER));
```

Comment:
Don't pass the class dependencies to the method params, it is better to use the constructor